### DISCUSSION TITLE
H3

### DISCUSSION THREAD

**Discussion by MattiasLindahl** [nathanmarlor/foxess_modbus#63] [General]
Hello!
Do you have any plans to make a version for the H3 models?

**Comment by nathanmarlor** [comment]
Yep, @benbeton is on it :)

https://github.com/nathanmarlor/foxess_modbus/issues/6

**Comment by canton7** [comment]
I've created the pre-release v1.4.1b3, which (in theory) adds support for H3 inverters. It's untested as I don't have access to a H3, so please give it a spin and let me know if you find any issues.
